src/Compilers/CSharp/Test/Semantic/Semantics/Utf8StringsLiteralsTests.cs

```diff
@@ -3384,6 +3384,77 @@ static void Main()
                 );
         }
 
+        [ConditionalFact(typeof(CoreClrOnly))]
+        public void VariableIsNotUsed()
+        {
+            var source = @"
+using System;
+class C
+{
+    static void Main()
+    {
+        const string stringNull = null;
+
+        var x1 = ""123""U8;
+        Span<byte> x2 = ""124""U8;
```

Comment:
Is it intentional that the warning is not given here even though it is given on 'z2'?

Reply:
> Is it intentional that the warning is not given here even though it is given on 'z2'?

This is similar to ```System.ReadOnlySpan<char> x3 = "abc";```. A user defined conversion considered as a usage.
 On `z2` there is no user-defined conversion from the language point of view, there is a built-in conversion.